src/panels/config/integrations/integration-panels/zwave/ha-config-zwave.js

```diff
@@ -301,7 +301,7 @@ class HaConfigZwave extends LocalizeMixin(EventsMixin(PolymerElement)) {
                   </paper-checkbox>
                   <paper-input
                     disabled="{{entityIgnored}}"
-                    label="[[localize('ui.panel.config.zwave.node_management.pooling_intensity')]]"
+                    label="[[localize('ui.panel.config.zwave.node_management.polling_intensity')]]"
```

Comment:
This will break all translations, I think it better not to fix this typo as no user will ever see it.

Reply:
Yeah, hence my reservations in the proposed change section. I'll close this PR.